### DISCUSSION TITLE
3.5.0 (20027) Beta 1

### DISCUSSION THREAD

**Discussion by Jason-Morcos** [Sequel-Ace/Sequel-Ace#1363] [General]
## Welcome to Sequel Ace 3.5!

Sequel Ace 3.5 is our next jam-packed update with new features from the community such as the ability to pin tables, filtering and sorting for the table structure view, enhanced support for default values, and much more! We greatly appreciate community help beta testing this new release and reporting any found issues. Happy querying!

---

### Added
- 2f33050, Fixed a crash with editing columns with non-string data
- 49ed5429, Support for pinning tables at top of tables list 
- 18af48c0,  Clicking S[tructure] C[ontent] or D[rop] in export window will toggle the value for all tables
- 7293fc50,   Filter and Sorting to Structure "fields" Table View.

### Changed
- da43e08c,  Accurately show SSL connection status always
- a37b653c,  Allow sorting every column on the Structure "fields" table view
- b002cae1,  Change default query editor font to Menlo to improve query editor performance

### Infra

<hr /><em>This discussion was created from the release <a href='https://github.com/Sequel-Ace/Sequel-Ace/releases/tag/beta/3.5.0-20027'>3.5.0 (20027) Beta 1</a>.</em>

**Comment by Visagalis** [comment]
Testing Pin tables functionality and it works great in all scenarios which I tried, very great addition by @ShashwatC 🙇 !

**Comment by n9yty** [comment]
For some reason this beta will not launch on my system. Must be from an old beta, I had to manually find and remove:

~/Library/Containers/com.sequel-ace.sequel-ace-beta/

**Reply by Jason-Morcos** [reply]
When you say "would not launch", does it crash on launch? Can you pull the crash from the console.app and share so we can see why it won't launch?

**Reply by n9yty** [reply]
See above, must have been left over from an older beta. ~/Library/Containers/com.sequel-ace.sequel-ace-beta/ was the culprit, once removed it launched.

**Reply by Jason-Morcos** [reply]
Right, but that doesn't really help me actually fix the issue :)
If you open console.app, are there crashes from Sequel Ace Beta from before you deleted that directory? Deleting the whole container basically makes the beta a "fresh" install, and I'm glad that fixed it, but that just means one of many many things could have caused it to not open. Usually a failure to launch should be paired with some sort of crash somewhere in the console.app

**Reply by n9yty** [reply]
Digging into the log command as console.app won't seemingly let me go back in time...  Hopefully this is helpful.

That did not preserve formatting, will try a file attachment.

[n9yty-sequel-ace-beta-crash.txt](https://github.com/Sequel-Ace/Sequel-Ace/files/7936595/n9yty-sequel-ace-beta-crash.txt)

**Reply by Jason-Morcos** [reply]
Thanks for finding that! Very very very weird, it looks like somehow the directory got fully corrupted. This might be a case where the only solution really is what you did - deleting the directory and starting over
